src/rgw/rgw_rest_swift.h

```diff
@@ -185,6 +185,29 @@ class RGWBulkDelete_ObjStore_SWIFT : public RGWBulkDelete_ObjStore {
   void send_response();
 };
 
+class RGWInfo_ObjStore_SWIFT : public RGWInfo_ObjStore {
+protected:
+  struct info
+  {
+    bool is_admin_info;
+    void (*list_data)(Formatter& formatter,
+                              md_config_t& config,
+                              RGWRados& store);
```

Comment:
looks like `config` can be passed as a const ref. `store` _should_ be a const ref, but it doesn't look like there's a const version of `RGWRados::get_zonegroup()`